modules/oxide/src/less/theme/components/tree/tree-button.less

```diff
@@ -198,6 +190,16 @@
     display: flex;
     flex-direction: column;
 
+    &.tox-tree--directory--expanded {
+      > .tox-tree--directory__label .tox-chevron {
+        transform: rotate(90deg);
+      }
+
+      > .tox-tree--directory__children {
+        max-height: 200px;
```

Comment:
Should this value change to `initial` instead of `200px` which is arbitrary ?

Reply:
I wanted to animate the expanding/collapsing of directory children but I found a better way using the `Sliding` behavior